.github/workflows/test.yml

```
@@ -99,16 +99,47 @@ jobs:
+    needs: [test]
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    continue-on-error: true
     permissions:
       contents: read
       pull-requests: write
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
         package: ${{ fromJson(needs.setup.outputs.packages) }}
```

actionlint (1.7.12)

[error] property "setup" is not defined in object type {test: {outputs: {}; result: string}}

(expression)